src/cluster_undersample/cluster_undersample.jl

```diff
@@ -0,0 +1,207 @@
+"""
+Assuming that all the observations in the observation matrix X belong to the same class, undersample
+the observations using clustering undersampling
+
+# Arguments
+- `X`: A matrix where each column is an observation of floats
+- `n`: Number of observations in the undersampled dataset
+- `mode::AbstractString="nearest`: If `center` then the undersampled data will consist of the centriods of 
+    each cluster found. Meanwhile, if `nearest` then it will consist of the nearest neighbor of each centroid.
```

Comment:
Do you mean:

```suggestion
- `mode::AbstractString="nearest`: If `"center"` then the undersampled data will consist of the centriods of 
    each cluster found. Meanwhile, if `"nearest"` then it will consist of the nearest neighbor of each centroid.
```

Reply:
Alternatively, it's probably more Julian to use a `Symbol` instead of a string.

Reply:
Yes, indeed I should have used quotes instead of backticks.
> Alternatively, it's probably more Julian to use a Symbol instead of a string.

Thanks, was not aware of that. I may consider changing all string arguments in all methods to symbols; if I don't then I think it still is more friendly to use strings for Julia beginners as this construct does not naturally exist in other languages.